tutorials/3d/portals/editing_rooms_and_portals.rst

```diff
@@ -21,46 +21,83 @@ Putting all the ideas together, here is an example scene tree:
+- ``kitchen*anything_you_want_here-portal`` - also works.
+
+Wildcards work on all of the nodes which use these naming conventions.
+
+Manual Bounds
+^^^^^^^^^^^^^
+Manual bounds are a way of explicitly setting the convex hull for a room, and are used if they are present as children of a room in the scene tree. Aside from the postfix, the naming is unimportant. They should be meshes (i.e. MeshInstance in Godot), and bear in mind they will be converted to convex hulls during the conversion process, so don't have to be perfect.
+
+.. tip:: Once used during conversion, they will be converted to ``portal_mode`` IGNORE and not shown. You can alternatively use ``Generate Points`` within the editor to convert these to a set of points stored in the room, and delete the original ``-bound`` MeshInstance.
+
 Portal point editing
 ~~~~~~~~~~~~~~~~~~~~
 
 Portals are defined by a combination of the transform of the portal node, and by a set of points which form the corners.
 
 The default portal has 4 corners as shown in the inspector:
 
-.. image:: img/portal_points.png
+.. image:: img/portal_point_editing.png
+
+You can edit these points in the gizmo or inspector to make a better match to the opening in your game level. It is generally better to keep the number of points as low as possible for the efficiency of the system. It is better to risk rendering a little too much than to spend the time culling objects at 20 different edges, for example.
 
-You can edit these points in the inspector to make a better match to the opening in your game level. But bear in mind it generally better to keep the number of points as low as possible for the efficiency of the system. It is better to risk rendering a little too much than to spend the time culling objects at 20 different edges, for example.
+.. _doc_room_point_editing:
 
 Room point editing
 ~~~~~~~~~~~~~~~~~~
 
-You also have the option to manually edit the points used to define the convex hull of a room. These points are not present by default, you would typically create them by pressing the **Generate Points** button in the editor toolbar, when a room is selected. This will transfer the auto bound from the geometry (or manual ``Bound_`` mesh) into the inspector. Once there are points in the inspector, they will be used and override any other method. So if you wish to revert your manual editing, simply delete all the points.
+You also have the option to manually edit the points used to define the convex hull of a room. These points are not present by default, you would typically create them by pressing the **Generate Points** button in the editor toolbar, when a room is selected. This will transfer the auto bound from the geometry (or manual ``-bound`` mesh) into the inspector. Once there are points in the inspector, they will be used and override any other method. So if you wish to revert your manual editing, simply delete all the points.
```

Comment:
```suggestion
You also have the option to manually edit the points used to define the convex hull of a room. These points are not present by default. You would typically create them by pressing the **Generate Points** button in the editor toolbar when a room is selected. This will transfer the auto bound from the geometry (or manual ``-bound`` mesh) into the inspector. Once there are points in the inspector, they will be used and override any other method. So if you wish to revert your manual editing, delete all the room's points.
```